src/pages/Search.tsx

```diff
@@ -276,6 +298,13 @@ const columns: Column<SearchResultTableEntry>[] = [
     Header: "Labware Type",
     accessor: "labwareType",
   },
+  {
+    Header: "SGP Numbers",
+    Cell: (props: Cell<SearchResultTableEntry>) => {
+      const workNumbers = props.row.original.workNumbers;
+      return workNumbers.join(", ");
+    },
+  },
```

Comment:
I think it is better to have this column as sortable because there was a previous requirement to make all columns in search results to be sortable

Reply:
@seenanair Its going to be a little tricky to sort because worknumbers column can have multiple work numbers, so not sure what the best way to sort is? I can apply the same sort as location uses maybe?

Reply:
@BenTopping Worknumber sort is implemented  in Dashboard page table. Would you please check that out?